windows/src/developer/TIKE/Tike.dpr

```diff
@@ -314,8 +314,8 @@ begin
       if TikeActive then Exit;
       InitClasses;
       Application.CreateForm(TmodWebHttpServer, modWebHttpServer);
-      Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
-      Application.Run;
+  Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
+  Application.Run;
```

Comment:
This indentation looks off?

Reply:
Delphi automatically updates this section of code -- indentation changes happen a lot so I don't tend to worry about them 😢